core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3280,6 +3280,16 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_METADATA_SYNC_AVOID_SAME_PATH_CONCURRENT_SYNC =
```

Comment:
I'm a bit unsure if this feature should be under a property key or not. If yes, if it should be default to true or false.

Reply:
cc @tcrain @yuzhu wdyt?

Reply:
MASTER_METADATA_SYNC_DEDUP?